src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/ProPlanControls/ProPlanControls.spec.jsx

```diff
@@ -587,21 +708,138 @@ describe('ProPlanControls', () => {
 
       it('renders the update button', async () => {
         setup({ planValue: Plans.USERS_PR_INAPPM })
-        render(<ProPlanControls />, { wrapper: wrapper() })
+        render(<ProPlanControls {...props} />, { wrapper: wrapper() })
 
         const update = await screen.findByRole('button', {
           name: /Update/,
         })
         expect(update).toBeInTheDocument()
       })
 
+      describe('when updating to a yearly plan', () => {
+        it('has the price for the year', async () => {
+          const { user } = setup({ planValue: Plans.USERS_PR_INAPPM })
+          render(<ProPlanControls {...props} />, { wrapper: wrapper() })
+
+          const annualOption = await screen.findByRole('button', {
+            name: 'Annual',
+          })
+          await user.click(annualOption)
+
+          const price = screen.getByText(/\$100/)
+          expect(price).toBeInTheDocument()
+        })
+      })
+
+      xdescribe('when the user has team plans available', () => {
```

Comment:
What is `xdescribe`?

Reply:
See this, https://github.com/codecov/gazebo/pull/2415#discussion_r1403754578

Reply:
Can you swap this to `describe.skip`? I feel like that's a bit more verbose and easier to find via fuzzy search.